runtime/fundamentals/stability_and_releases.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Stability and releases"
+oldUrl:
+    - /runtime/manual/runtime/stability/
+    - /runtime/fundamentals/stability/
+---
+
+As of Deno 1.0.0, the `Deno` namespace APIs are stable. That means we will
+strive to make code working under 1.0.0 continue to work in future versions.
+
+When introducing new APIs, these are first marked as unstable. This means that
+the API may change in the future. These APIs are not available to use unless you
+explicitly pass an unstable flag, like `--unstable-kv`.
+[Learn more about `--unstable-*` flags](/runtime/reference/cli/unstable_flags).
+
+There are also some non-runtime features of Deno that are considered unstable,
+and are locked behind unstable flags. For example, the
+`--unstable-sloppy-imports` flag is used to enable `import`ing code without
+specifying file extensions.
+
+## Release schedule, channels and long term support
+
+Deno releases a new stable, minor version (eg. v2.1.0, v2.0.0) on a monthly
+schedule.
```

Comment:
Could add ...mostly inline with Chrome V8 releases

Reply:
I think this information is irrelevant to an average Deno user.